content/desktop/hardened-desktop/enhanced-container-isolation/limitations.md

```diff
@@ -82,8 +82,15 @@ these won't work properly.
 
 ### Kubernetes pods are not yet protected
 
-Kubernetes pods are not yet protected by ECI. A malicious or privileged pod can
-compromise the Docker Desktop Linux VM and bypass security controls.
+When using the Docker Desktop integrated Kubernetes, pods are not yet protected
+by ECI. Therefore a malicious or privileged pod can compromise the Docker
+Desktop Linux VM and bypass security controls.
+
+As an alternative, you can use the [K8s.io KinD](https://kind.sigs.k8s.io/) tool
+with ECI. In this case, each Kubernetes node runs inside an ECI-protected
+container, thereby more strongly isolating the Kubernetes cluster away from the
+underlying Docker Desktop Linux VM (and Docker engine within). No special
```

Comment:
Looks like the linter prefers `Docker Engine` (capital `E`)

Reply:
Done, thanks!